src/charm.py

```diff
@@ -73,7 +114,7 @@ def _pebble_checks(self) -> Dict[str, Any]:
                 "override": "replace",
                 "level": "alive",
                 "period": "30s",
-                "http": {"url": "http://localhost:13133/health"},
+                "http": {"url": f"http://localhost:{Ports.HEALTH.value}/health"},
```

Comment:
```suggestion
                "http": {"url": f"http://localhost:{Ports.HEALTH.value}/health"},  # TODO: TLS
```
Or just remember to search for "http:" when adding TLS :)